defaults/main.yml

```diff
@@ -52,4 +60,34 @@ tiaas_secret_key: "Using the default is bad news bears"
 tiaas_admin_user: admin
 tiaas_admin_pass: admin
 
-tiaas_show_advertising: true
+# Add these lines to playbook vars to override default tiaas files with
+# playbook files:
+
+# tiaas_templates_dir: files/tiaas/html  # < copy templates from <playbook>/files/html
+# tiaas_extra_static_dir: files/tiaas/static  # < put static files required by above templates in here
+
+tiaas_html_templates:
+  - src: "footer.html"
+    dest: "{{ tiaas_code_dir }}/templates/snippets/footer.html"
```

Comment:
They're "templates", so, let's put them in the template folder so they can use it specific variables.

Reply:
@hexylena tried and failed... the Django template tags are interpretted as Jinja tags by Ansible! We could use `{% raw %}` but it would get real messy/confusing. Probably better to copy as files.

Reply:
Ahhh boo, of course, ah well.